internal/addon/options.go

```diff
@@ -9,19 +11,23 @@ const (
 	KeyOpenShiftLoggingChannel = "openshiftLoggingChannel"
 
 	// Platform Observability Keys
-	KeyPlatformLogsCollection = "platformLogsCollection"
+	KeyPlatformMetricsCollection = "platformMetricsCollection"
+	KeyPlatformLogsCollection    = "platformLogsCollection"
+	KeyMetricsHubHostname        = "metricsHubHostname"
 
 	// User Workloads Observability Keys
-	KeyUserWorkloadLogsCollection   = "userWorkloadLogsCollection"
-	KeyUserWorkloadTracesCollection = "userWorkloadTracesCollection"
-	KeyUserWorkloadInstrumentation  = "userWorkloadInstrumentation"
+	KeyUserWorkloadMetricsCollection = "userWorkloadMetricsCollection"
+	KeyUserWorkloadLogsCollection    = "userWorkloadLogsCollection"
+	KeyUserWorkloadTracesCollection  = "userWorkloadTracesCollection"
+	KeyUserWorkloadInstrumentation   = "userWorkloadInstrumentation"
 )
 
 type CollectionKind string
 
 const (
-	ClusterLogForwarderV1         CollectionKind = "clusterlogforwarders.v1.observability.openshift.io"
-	OpenTelemetryCollectorV1beta1 CollectionKind = "opentelemetrycollectors.v1beta1.opentelemetry.io"
+	ClusterLogForwarderV1                   CollectionKind = "clusterlogforwarders.v1.observability.openshift.io"
+	OpenTelemetryCollectorV1beta1           CollectionKind = "opentelemetrycollectors.v1beta1.opentelemetry.io"
+	PrometheusAgentMetricsCollectorV1alpha1 CollectionKind = "prometheusagents.v1alpha1.monitoring.coreos.com"
```

Comment:
WDYT of simplifying the name here
```suggestion
	PrometheusAgentV1alpha1 CollectionKind = "prometheusagents.v1alpha1.monitoring.coreos.com"
```